services/fastmail/config.json

```diff
@@ -19,16 +27,114 @@
       "prio": 20
     },
     {
+      "name": "*",
+      "type": "MX",
+      "content": "in1-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 10
+    },
+    {
+      "name": "*",
+      "type": "MX",
+      "content": "in2-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 20
+    },
+    {
+      "name": "mail",
+      "type": "MX",
+      "content": "in1-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 10
+    },
+    {
+      "name": "mail",
+      "type": "MX",
+      "content": "in2-smtp.messagingengine.com",
+      "ttl": 3600,
+      "prio": 20
+    },    {
       "name": "mail",
       "type": "CNAME",
```

Comment:
This record would need to become an `ALIAS` to continue working properly. A `CNAME` can be the only record on a zone, so using an `ALIAS` here would allow both the `MX` records and this one to resolve properly.